app/src/main/java/com/futurice/freesound/feature/search/DefaultSearchDataModel.java

```diff
@@ -38,7 +38,7 @@
     private final FreeSoundSearchService freeSoundSearchService;
 
     @NonNull
-    private final BehaviorSubject<Option<List<Sound>>> lastResults = BehaviorSubject.create();
+    private final BehaviorSubject<Option<List<Sound>>> lastResultsStream = BehaviorSubject.create();
```

Comment:
OK. There's probably a case for it having a default value and changing the tests, but this PR is not the place for it anyway.